<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSeriesControl</class>
 <widget class="QWidget" name="ImageSeriesControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="camera_folder_edit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="camera_folder_choose_button">
     <property name="text">
      <string>Choose...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>File Name Root</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="camera_filename_root_edit">
     <property name="text">
      <string>camera</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_28">
     <property name="text">
      <string>Number of Images</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="buffer_size_spin">
     <property name="maximum">
      <number>5000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="camera_streaming_button">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
